doc/template.hbs

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+{{!-- Based on https://github.com/Redocly/redoc/blob/2100b549c0f1d54c95c41fd0d3eb0b6b23deccb1/cli/template.hbs --}}
+<html>
+
+<head>
+  <meta charset="utf8" />
+  <title>{{title}}</title>
+  <!-- needed for adaptive design -->
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <style>
+    body {
+      padding: 0;
+      margin: 0;
+      word-break: break-word;
```

Comment:
This is the relevant (and only adjusted) line. Rest is the same as the default template.